src/internal/connector/onedrive/restore_test.go

```diff
@@ -0,0 +1,152 @@
+package onedrive
+
+import (
+	"testing"
+
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/internal/version"
+	"github.com/alcionai/corso/src/pkg/path"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type RestoreUnitSuite struct {
+	tester.Suite
+}
+
+func TestRestoreUnitSuite(t *testing.T) {
+	suite.Run(t, &RestoreUnitSuite{Suite: tester.NewUnitSuite(t)})
+}
+
+func (suite *RestoreUnitSuite) TestAugmentRestorePaths() {
+	table := []struct {
+		name    string
+		version int
+		input   []string
+		output  []string
+	}{
+		{
+			name:    "no change v0",
+			version: 0,
+			input: []string{
+				"file.txt.data",
```

Comment:
just for my paranoia can you add an additional file that has no `.data` suffix to this test case and the v0 test case below? That would help check that
1. the code doesn't depend on the `.data` suffix in some way
2. we give it input without the `.data` suffix which is what we'll get in v0 backups in most cases (unless the user has something with `.data`)

This is totally one of those "what ways do the tests assume the code may be broken" sort of things that I get paranoid about 😅

Reply:
Ahh, good catch!